src/RadworksGovernor.sol

```diff
@@ -174,11 +178,17 @@ contract RadworksGovernor is
     return GovernorTimelockCompound.state(proposalId);
   }
 
-  /// @notice The amount of RAD required to meet the quorum threshold for a proposal
-  /// as of a given block.
+  /// @notice The amount of RAD required to meet the quorum threshold for a proposal.
   /// @dev Our implementation ignores the block number parameter and returns a constant.
-  function quorum(uint256) public pure override returns (uint256) {
-    return QUORUM;
+  function quorum(uint256) public view override returns (uint256) {
+    return _quorum;
+  }
+
+  /// @notice Set the amount of RAD required to meet the quorum threshold for a proposal.
+  /// @dev This function is only callable by executed governance proposals (thus, the Timelock).
+  function setQuorum(uint256 _newQuorum) public onlyGovernance {
+    emit QuorumSet(_newQuorum);
```

Comment:
It may be good to check this event is emitted in the test.

Reply:
Done: 73f73c9c7237d16400c23d610d559f3ef7563e40